build-system/tasks/bundle-size/README.md

```diff
@@ -0,0 +1,23 @@
+# Bundle Size checks
+
+The `APPROVERS.json` file is a listing of compiled AMP runtime and extensions
+files. Each rule has two parts:
+
+- `approvers`: list of GitHub project teams, members of which are allowed to
+  approve pull requests that fail the check
+- `threshold`: number of KBs that the file bundle size (Brotli-compressed size)
+  can increase before failing the check
+
+Approval from any single member of any of the file's approval-teams is enough to
+satisfy the check.
+
+This file is dynamically fetched by the [Bundle-Size GitHub App](https://github.com/ampproject/amp-github-apps/tree/master/bundle-size)
+
+Note that pull requests that increase the bundle size of multiple files that
+each require approval by a different set of teams (e.g., a PR that increases the
+bundle-size of three files, requiring approvals by `{team1}`, `{team1, team2}`,
+and `{team2}`) will fall back to the set of default approvers, currently set to
+@ampproject/wg-runtime and @ampproject/wg-performance. Members of
```

Comment:
Does this happen when two different files increase in size? Or only when two files with different `approvers` lists increase in size?

Either way, I think the first sentence can be simplified. Something like:

> Note: If a PR requires a bundle-size approval (from more than one set of approvers) for more than one file, it will fall back to the default set of approvers (`@ampproject/wg-runtime and @ampproject/wg-performance`).

Reply:
Done